src/backend/columnar/cstore_tableam.c

```diff
@@ -1378,6 +1384,32 @@ columnar_handler(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * Verify that logical replication is not active on the relation.
+ */
```

Comment:
Minor: In Citus code we usually start the function comment with the function name.

```c
/*
 * ColumnarCheckLogicalReplication throws an error if the relation is part of
 * a publication that publishes inserts, since those are not yet replicated
 * through logical replication.
 */
```

Reply:
Thank you, fixed.